prac_05/emails.py

```diff
@@ -0,0 +1,21 @@
+"""
+Emails program
+Estimated: 50min
+Actual:
+"""
+
+email = input("Email: ")
+email_to_name = {}
+while email != "":
+    name = " ".join(email.split("@")[0].title().split("."))
+    is_name = input(f"Is your name {name}? (Y/n) ").lower()
+    if is_name == "y" or is_name == "":
+        email_to_name[email] = name
+    else:
+        # Name could be empty
```

Comment:
I think i approached this from the opposite, with name == 'n', and didn't account for anything else. I might have to go back and take a look!

Reply:
If you look at the requirement he explains that the program asks for the name if its wasn't Y or enter. It can be hard to miss.